### ISSUE TITLE
EDA plot() not showing properly in VScode

### ISSUE THREAD

**Issue by ldavidr3** [sfu-db/dataprep#952]
**Describe the bug**
EDA plot() not showing properly in VScode

**To Reproduce**
1. Open new Jupyter notebook
2. Import from dataprep.eda import plot, plot_correlation, create_report, plot_missing
3. Try .head() method on any test dataset so it shows first DF rows as output
4. Next cell call .plot() method from dataprep
5. Near finishing execution, the method  adds padding to the left of all Output cells and it's impossible to scroll to the right to see the full output

Or:

```python

import numpy as np
import pandas as pd
import datetime
from datetime import date
import matplotlib
import seaborn as sns
import matplotlib.pyplot as plt
import plotly.graph_objects as go
from sklearn.preprocessing import StandardScaler, normalize
from sklearn import metrics
from sklearn.mixture import GaussianMixture
from mlxtend.frequent_patterns import apriori
from mlxtend.frequent_patterns import association_rules
import warnings
warnings.filterwarnings('ignore')
data=pd.read_csv('marketing_campaign.csv',header=0,sep=';')

from dataprep.eda import plot, plot_correlation, create_report, plot_missing
plot(data)
```

**Expected behavior**
I expected to be shown the output normally in the screen

**Screenshots**

Images: 
![image_2022-12-20_105546368](https://user-images.githubusercontent.com/100173574/208697030-86c02c90-b097-48e8-b3c5-efd262a30807.png)

![image_2022-12-20_105618462](https://user-images.githubusercontent.com/100173574/208697155-4ffa66d5-af83-4fc6-8425-5219b90863d3.png)

![image_2022-12-20_105702601](https://user-images.githubusercontent.com/100173574/208697327-09f50e2e-7903-4189-adfe-c64b297b178b.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: None
 - Platform VSCode 
 - Platform Version 1.74.1
 - Python Version 3.10.9
 - Dataprep Version 0.4.4

**Additional context**

**Comment by dovahcrow** [comment]
Nice catch! The visualization is implemented/tested for jupyter notebook at the very beginning. With the popularity of VSCode increasing, I think supporting it should be the next step. Will circle around the issue with the team to see what exactly happens.

**Comment by yimin-demo** [comment]
I have the same issue when I run it in VSCode. Could I know which plot package you use to generate plots? Probably I can try to search for if the issue is from the plot function itself. Thanks.

**Comment by Krysalide** [comment]
Same issue on vscode, i have seen a config dict in plot function. is there a way to use config to get rid of the left padding?

**Comment by mib1213** [comment]
Is there still no solution to this problem?
